plugins/woocommerce/includes/admin/class-wc-admin-settings.php

```diff
@@ -458,6 +463,9 @@ class="<?php echo esc_attr( $value['class'] ); ?>"
 										</li>
 										<?php
 									}
+									if ( $show_desc_at_end ) {
+										echo wp_kses_post( "<p class='description description-thin'>{$value['desc']}</p>" );
```

Comment:
(2/2) If the description is shown at the end, we echo `$value['desc']`. Even if these are effectively aliases, seems it might be better to use the same one in both places?

Reply:
Unless this is deliberate, of course.